### PR TITLE
S2 display teams t4

### PR DESCRIPTION
**Pull request by myc-design**
Peer review request

### CODE REVIEW

File: app/js/app.js [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log('test');
```
Comment (richcatt) [inline app/js/app.js]: What's this for? Can we delete it?

File: index.php [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+        <div class="content">
+            <img src="https://dev.maydenacademy.co.uk/resources/sports_teams/canucks.png" />
+                <ul>
+                    <li>Sport: Football</li>
+                    <li>Country: UK</li>
+                    <li>Team colours: Red/Yellow</li>
+                </ul>
+        </div>
+    </section>
+    <section>
```
Comment (richcatt) [inline index.php]: Can you check the indentation here please?

File: index.php [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+    <link type='text/css' rel='stylesheet' href='./app/css/normalize.css' />
+    <link type='text/css' rel='stylesheet' href='./app/css/style.css' />
+</head>
+<body>
+<?php require_once './vendor/autoload.php'; ?>
+<header>
+    <h1>The Real MVP</h1>
+</header>
+<form action="#" method="GET">
+<main>
```
Comment (richcatt) [inline index.php]: Can we not have main inside forms please

File: index.php [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+    <link type='text/css' rel='stylesheet' href='./app/css/style.css' />
+</head>
+<body>
+<?php require_once './vendor/autoload.php'; ?>
+<header>
+    <h1>The Real MVP</h1>
+</header>
+<form action="#" method="GET">
+<main>    
+<section>
```
Comment (richcatt) [inline index.php]: Can we use something else instead of section? E.g. article?

File: src/GetAPI.php [hunk 5]
```diff
@@ -2,7 +2,6 @@
```
Comment (richcatt) [inline src/GetAPI.php]: Isn't this a separate task branch?

**Review by richcatt — CHANGES_REQUESTED** [review]

File: src/GetAPI.php (hunk 5 above)
Comment (benjaminrshill) [reply src/GetAPI.php]: yeees we branched off the task branch :/

**Review by richcatt — APPROVED** [review]

**Pull request merged**